source/puppet/5.3/hiera_automatic.md

```diff
@@ -6,20 +6,20 @@ title: "Looking up data with Hiera"
 [data_type]: ./lang_data_type.html
 [editing_data]: ./hiera_merging.html
 
-{:.Concept}
```

Comment:
These classes for the DITA converter are case-sensitive and should be lowercased.